deploy/charts/rook-ceph/templates/resources.yaml

```
@@ -2856,7 +2856,7 @@ spec:
                       type: string
                       x-kubernetes-validations:
                         - message: network provider must be disabled (reverted to empty string) before a new provider is enabled
-                          rule: self == '' || self == oldSelf
+                          rule: oldSelf == '' || self == oldSelf
```

@subhamkrai @BlaineEXE Do we need both `self` and `oldSelf` to be allowed to be blank to convert either direction?
```suggestion
                          rule: self == '' || oldSelf == '' || self == oldSelf
```

---

That's what I was thinking as well

---

Thanks for the review.
Good catch, that makes sense, let me fix that.